src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -4669,4 +4669,4 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_InvalidReal" xml:space="preserve">
     <value>Invalid real literal.</value>
   </data>
-</root>
+</root>
```

Comment:
Ah, the IDE does that I remember (though VS2015 does respect linebreak sequence in case of csproj file, which down versions used to overwrite..). This means that previously someone put the linebreak by hand so probably a good idea to continue keeping it (and keep the diffs bit cleaner) IMO..